example/user/user.pb.gorm.go

```diff
@@ -123,18 +125,20 @@ func (m *User) ToORM() (UserORM, error) {
 		}
 	}
 	if posthook, ok := interface{}(m).(UserWithAfterToORM); ok {
-		posthook.AfterToORM(&to)
+		err = posthook.AfterToORM(ctx, &to)
 	}
 	return to, err
 }
 
 // FromORM returns a pb object
```

Comment:
not directly related to this PR, but it looks like this comment is no longer generating the proper name